frontend/src/bundles/studio/components/timeline/row.tsx

```diff
@@ -0,0 +1,26 @@
+import { type RowDefinition, useRow } from 'dnd-timeline';
+
+import { Box } from '~/bundles/common/components/components.js';
+
+type Properties = RowDefinition & {
+    children: React.ReactNode;
+};
+
+const Row = (properties: Properties): JSX.Element => {
```

Comment:
It is better to use destructuring since it was used before. Stick to one code style

Reply:
Thank you. I thought that since the properties only have one element, it wasn't necessary. I'll commit to the destructuring approach from now on. Thank you

Reply:
Solved. Thank you